keras_nlp/metrics/bleu.py

```diff
@@ -92,7 +92,86 @@ class Bleu(keras.metrics.Metric):
         dtype: string or tf.dtypes.Dtype. Precision of metric computation. If
                not specified, it defaults to tf.float32.
         name: string. Name of the metric instance.
-        **kwargs: Other keyword arguments.
+
+    Examples:
+
+    1. Various Input Types.
+    1.1. Python string.
+    >>> bleu = keras_nlp.metrics.Bleu(max_order=4)
+    >>> ref_sentence = "the quick brown fox jumps over the lazy dog"
+    >>> pred_sentence = "the quick brown fox jumps over the box"
+    >>> score = bleu([ref_sentence], [pred_sentence])
+    <tf.Tensor(0.7420885, shape=(), dtype=float32)>
+
+    1.2. rank 1 inputs.
```

Comment:
We should change these to 

```
rank 1 prediction inputs, rank 2 reference inputs
```

or something like that?

Reply:
I checked the Rouge L metric documentation, where this same convention is used. Is it okay to make these changes?